scripts/lib/zulip_tools.py

```diff
@@ -336,15 +336,26 @@ def parse_lsb_release():
                 continue
             distro_info[k] = v
     except FileNotFoundError:
-        # Unfortunately, Debian stretch doesn't yet have an
-        # /etc/lsb-release, so we instead fetch the pieces of data
-        # that we use from the `lsb_release` command directly.
-        vendor = subprocess_text_output(["lsb_release", "-is"])
-        codename = subprocess_text_output(["lsb_release", "-cs"])
-        distro_info = dict(
-            DISTRIB_CODENAME=codename,
-            DISTRIB_ID=vendor
-        )
+        try:
+            # Might be a centos
+            with open('/etc/centos-release', 'r') as fp:
+                info = fp.read().strip().split(' ')
+            vendor = info[0]
+            codename = 'centos' + info[3][0]
+            distro_info = dict(
+                DISTRIB_CODENAME=codename,
+                DISTRIB_ID=vendor
+            )
+        except FileNotFoundError:
+            # Unfortunately, Debian stretch doesn't yet have an
```

Comment:
Debian strech has `/etc/os-release`.

Reply:
I think rather than chaining exception handlers, we should just have at the top an early-return block of the form:
```
if os.path.exists("/etc/redhat-release"):
   ...
   return distro_info
```

And then not need to patch the existing Debian/Ubuntu code.

BTW, we should look at `/etc/redhat-release`; that exists on RHEL as well as CentOS and I believe has the same format.  So we'd potentially get RHEL for free.